modules/user-provided-certificates-for-api-server.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// *authentication/certificate-types-descriptions.adoc
+
+[id="user-provided-certificates-for-the-api-server_{context}"]
+= User-provided certificates for the API server
+
+[discrete]
+== Purpose
+
+The API server is accessible by clients external to the cluster at
```

Comment:
"User-provided certificates " - are we talking here about "certificate-authority-data" from kubeconfig? or we are talking about 3rd party certificates we can add here?